example_agents/acme_dqn/agent.py

```diff
@@ -20,8 +20,12 @@ def __init__(self, **kwargs):
         )
 
     def evaluate(self, num_episodes):
-
-        with self.AcmeRun("evaluate", active_component_run(self)) as run:
+        with self.AcmeRun(
+            run_type="evaluate",
```

Comment:
This was my bad I think, but could you swap out these magic `run_type` strings in the example agents for the constants in the run class?

Reply:
Instead of making users import & use constants, which makes the API more cumbersome to users, I added two convenience factory class functions to AgentRun: `AgentRun.learn_run()` and `AgentRun.evaluate_run()`.